icc-api/api/iccUserApi.ts

```diff
@@ -87,22 +87,6 @@ export class iccUserApi {
       .then(doc => (doc.body as Array<JSON>).map(it => JSON.parse(JSON.stringify(it))))
       .catch(err => this.handleError(err))
   }
-  findByHcpartyId(id: string): Promise<Array<models.LabelledOccurenceDto> | any> {
-    let _body = null
-
-    const _url =
-      this.host +
-      "/user/byHealthcarePartyId/{id}".replace("{id}", id + "") +
-      "?ts=" +
-      new Date().getTime()
-    let headers = this.headers
-    headers = headers
-      .filter(h => h.header !== "Content-Type")
-      .concat(new XHR.Header("Content-Type", "application/json"))
-    return XHR.sendCommand("GET", _url, headers, _body)
-      .then(doc => (doc.body as Array<JSON>).map(it => new models.LabelledOccurenceDto(it)))
-      .catch(err => this.handleError(err))
-  }
```

Comment:
We created this function today together with @phil1618 , please regenerate genloc once antoinepairet/icure-backend is merged in taktik/icure-backend